core/src/apps/management/recovery_device/homescreen.py

```diff
@@ -232,17 +232,15 @@
             if not share:
                 share = slip39.decode_mnemonic(m)
             identifier = m.split(" ")[0:3]
-            identifiers.append([remaining, identifier])
+            item = (remaining, tuple(identifier))
+            groups.add(item)
```

Comment:
instead: `groups.add((remaining, tuple(identifier))`

Reply:
Hmm, I thought this won't work, since the parentheses will be considered as "dividers". Interesting, thanks.

Reply:
They are! But the expression `a, b, c` is a tuple itself.
So parentheses on `item = (a, b)` are superfluous, like `item = ("hello")`

Reply:
Aah, makes sense! Thanks.